mlflow/server/js/src/components/CompareRunScatter.js

```diff
@@ -0,0 +1,222 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { getParams, getRunInfo } from '../reducers/Reducers';
+import { connect } from 'react-redux';
+import './CompareRunView.css';
+import { RunInfo } from '../sdk/MlflowMessages';
+import Utils from '../utils/Utils';
+import { getLatestMetrics } from '../reducers/MetricReducer';
+import {ScatterChart, Scatter, XAxis, YAxis, CartesianGrid, Tooltip, ResponsiveContainer, Label} from 'recharts';
+import './CompareRunScatter.css';
+
+class CompareRunScatter extends Component {
+  static propTypes = {
+    runInfos: PropTypes.arrayOf(RunInfo).isRequired,
+    metricLists: PropTypes.arrayOf(Array).isRequired,
+    paramLists: PropTypes.arrayOf(Array).isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.metricKeys = CompareRunScatter.getKeys(this.props.metricLists);
+    this.paramKeys = CompareRunScatter.getKeys(this.props.paramLists);
+
+    if(this.paramKeys.length + this.metricKeys.length < 2)
+      this.state = {disabled: true};
+    else
```

Comment:
Put curly braces around if and else bodies even if they have just one statement, to follow the existing code style. This can lead to really confusing issues if someone wants to add a line of code in one of the bodies.